llm/gemma/serve.yaml

```diff
@@ -0,0 +1,48 @@
+# A example yaml for serving Gemma model from Mistral.ai with an OpenAI API.
+# Usage:
+#  1. Launch on a single instance: `sky launch -c gemma ./serve.yaml`
+#  2. Scale up to multiple instances with a single endpoint:
+#     `sky serve up -n gemma ./serve.yaml`
+service:
+  readiness_probe:
+    path: /v1/chat/completions
+    post_data:
+      model: $MODEL_NAME
+      messages:
+        - role: user
+          content: Hello! What is your name?
+      max_tokens: 1
+    initial_delay_seconds: 1200
+  replica_policy:
+    min_replicas: 2
+
+envs:
+  MODEL_NAME: google/gemma-7b
+  HF_TOKEN: <your-huggingface-token> # TODO: Replace with huggingface token
+
+resources: 
```

Comment:
Azure is taking a long time without success (quota failover), so I changed this to
```
resources:
  accelerators: {L4, A10g, A10, L40, A40, A100, A100-80GB}
  ports: 8000
  disk_tier: best
  any_of:
    - cloud: aws
    - cloud: gcp
```
and got
```

  File "/Users/zongheng/Dropbox/workspace/riselab/sky-computing/sky/serve/core.py", line 88, in up
    raise ValueError(f'Got multiple clouds: {requested_cloud} and '
ValueError: Got multiple clouds: GCP and AWS in different resources. Please specify single cloud instead.
```
for `sky serve up`. Is this expected?

Reply:
This seems to be an issue that needs to be fixed @cblmemo @MaoZiming ?

Reply:
Good catch! Fixed in #3226. Thanks!